node/nodejs_intro/README.md

```diff
@@ -69,55 +102,78 @@ Let's take a look at the official Chrome V8 website: [https://developers.google.
 > 7 or later, macOS 10.5+, and Linux systems that use IA-32, ARM, or MIPS processors. V8 can run standalone, or can be 
 > embedded into any C++ application. More information can be found on V8's public wiki.
 
-This is basically Google's open source engine that runs and interprets Javascript on all our Google Chrome web browsers.
+This is the specific browser tool that Google uses to run and interpret Javascript on all our Google Chrome web browsers.
 
-The very smart people at Google then thought: What if we just run this standalone without a browser on a computer? That's how Node.js was born.
+But one day, the very smart people at Google thought: **What if we just run this V8 Javascript Engine on it's own without a browser on a computer? This way, we can use it to write server-side code using Javascript!**
```

Comment:
I just have a factual question here. Is the original statement correct that Google created Nodejs or that Nodejs was a Google project? 

https://en.wikipedia.org/wiki/Node.js#:~:text=Node.js%20was%20written%20initially,and%20later%20sponsored%20by%20Joyent.

Reply:
Great catch, I think that is not true, Google did not make Node